mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/MasterMain.java

```diff
@@ -74,10 +81,15 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.HashSet;
+import java.util.Optional;
 import java.util.Properties;
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.function.Consumer;
+import org.apache.flink.configuration.Configuration;
```

Comment:
Is this going to be a short-term plan? I feel like this could cause some pain in the future upgrade if things changed on the flink side (I don't feel like this is something they will consider/maintain back-compat carefully though).